src/App.tsx

```diff
@@ -56,11 +84,14 @@ function App() {
               },
               validate: handlePasswordValidation,
             })}
+            onChange={(e) => setPassword(e.target.value)}
           />
 
           {errors.password && <p>{errors.password.message}</p>}
         </div>
 
+        <p>{loginError}</p>
```

Comment:
aqui a tag `p` vai sempre estar presente no html porem ela vai estar vazia, vc mostrar ela somente quando o loginError existir né.

```suggestion
        {loginError && <p>{loginError}</p>}
```